prow/config/external_plugins_config.yaml

```diff
@@ -1875,3 +1874,30 @@ ti-community-issue-triage:
     linked_issue_needs_triage_label: "do-not-merge/needs-triage-completed"
     need_cherry_pick_label_prefix: "needs-cherry-pick-release-"
     status_target_url: "https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/cherrypick-a-pr.html#pass-triage-complete-check"
+  - repos:
+      - tikv/tikv
+    maintain_versions:
+      - "4.0"
+      - "5.0"
+      - "5.1"
+      - "5.2"
+      - "5.3"
+      - "5.4"
+      - "6.0"
+      - "6.1"
+      - "6.2"
+      - "6.3"
+    wontfix_versions:
+      - "3.0"
```

Comment:
the item is not in `maintain_versions`, does it need to be contained in the `wontfix_versions` list? @MimeLyc